wikipedia-dark.user.css

```diff
@@ -3974,7 +4287,6 @@ regexp("https?:\/\/wikiless\.org\/.*$") {
   img[src*="Portal.svg"], img[src*="Information.svg"],
   img[src*="Global_thinking.svg"], img[src*="Clipboard.svg"],
   img[src*="Darkgreen_flag_waving.svg"], img[src*="Stock_post_message.svg"],
-  img[src*="Featured_article_star_-_cross.svg"],
```

Comment:
do not remove anything like this, it exists for a reason.

Reply:
It was overriding that USSR star, thats why I removed it.

Reply:
yes I understand, the correct way is to add a new entry that matches more precisely the entry to fix. Because the override above was too generic you can aslo do this...

`img[src*="Featured_article_star_-_cross.svg"]:not([src*="USSR"]` for instance...

The reason why this is preferred even though it maybe a little crazy to do, is because we support many many many different wikis both domains and languages which you may not be automatically aware and some overrides in main section are for the most, not the few.

The alternative is, to add specific overrides for a domain under its own section, for instance russian sites we have

@-moz-document domain("ru.wiktionary.org") and domain("ru.wikipedia.org"), domain("ru.wiktionary.org").

so say this was a ru site you would create an entry that matches the USSR cross and turn filter off.

e.g. 

``` css
img[src*="Featured_article_star_-_cross_USSR.svg"] {
  filter: none !important;
}
```